tests/models/model-1.json.orig

```diff
@@ -0,0 +1,201 @@
+{
```

Comment:
Thanks I have more changes to check in. Struggling with the rebate, I think I will take fresh copy from you and try again. 

Sent from my iPhone

> On 3 Feb 2015, at 16:30, William Candillon <notifications@github.com> wrote:
> 
> In tests/models/model-1.json.orig:
> 
> > @@ -0,0 +1,201 @@
> > +{
> Looks like it was committed by mistake.
> 
> —
> Reply to this email directly or view it on GitHub.
>